### ISSUE TITLE
Data structure image is confusing after category class switch CTN94-128

### ISSUE THREAD

**Issue by mbreevoort** [CTN-0094/DOPE#44]
In commit https://github.com/CTN-0094/DOPE/commit/a402e5269cab7422dc8a1e039e60240cde6c512d the class and category are switched. 
The image in the README with the data structure still shows for example `DEA category = stimulants` and `DEA class = amphetamines` which should be `DEA class = stimulants` and `DEA category = amphetamines`

Thanks for creating the opensource DOPE model. Because the image source is not the I cannot create I pull request, is there a way to fix this?

**Comment by RaymondBalise** [comment]
Thank you for pointing out the issue.  We have been teaching and could not address it.  I will get on it soon.

**Comment by labouz** [comment]
no, the image is created with a software called iThoughtsX. It could potentially be a `diagrammeR` or `grViz` or another  package that creates flow charts that I can't remember atm. We are open to collaboration!
